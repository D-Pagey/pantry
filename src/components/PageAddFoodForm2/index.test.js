import React from 'react';
import userEvent from '@testing-library/user-event';
import { PageAddFoodForm2 } from '.';

describe('PageAddFoodForm2 component', () => {
    it('should render', () => {
        const { container } = render(<PageAddFoodForm2 />);
        expect(container.firstChild).toMatchSnapshot();
    });

    it('should render step 2 of the form once category is clicked', () => {
        const { getByTestId, getByText } = render(<PageAddFoodForm2 />);

        userEvent.click(getByTestId('meatCategoryButton'));

        getByText('Step 2');
    });
});