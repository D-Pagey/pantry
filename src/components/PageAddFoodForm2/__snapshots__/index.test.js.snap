// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageAddFoodForm2 component should render 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: white;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,.06);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 33px 24px;
  left: 0;
  position: fixed;
  right: 0;
  top: 0;
}

.c1 {
  color: initial;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  font-size: 25px;
  font-weight: 500;
  margin: 0;
  padding: 0 0 0 14px;
}

.c2 {
  position: relative;
  top: 2px;
  width: 32px;
}

<div
  class="c0"
>
  <a
    aria-current="page"
    class="c1 active"
    href="/"
  >
    <img
      alt="icon"
      class="c2"
      src="icon.svg"
    />
  </a>
  <h4
    class="c3"
  >
    Pantry
  </h4>
</div>
`;
